File: Assignment_48/Q3.java
///////////////////////////////////////////////////////////////////////////////////////////////
//
//  File name :     Assignment 48-Q3.java
//  Description : write java program which accept string from user and display below pattern.
//  Author :        Vaishanvi D Shingare
//  Date :          2/6/2025

/* INPUT  : Hello
   OUTPUT : H 
            H e 
            H e l 
            H e l l
            H e l l o
*/
           
///////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Q2
{
    public void Pattern(String str) 
    {
        int ino = str.length();

        for (int i = 0; i <= ino; i++) 
        {
            
            for (int j = 0 ; j <= (i - 1); j++) 
            {
                System.out.print(str.charAt(j) + " ");
            }
            System.out.println();
        }
    }

    public static void main(String[] args) 
    {
        Scanner sobj = new Scanner(System.in);
        
        System.out.print("Enter a string: ");
        String input = sobj.nextLine();

        Q2 obj = new Q2();
        obj.Pattern(input);
    }
}

File: Assignment_40/Q3.c
///////////////////////////////////////////////////////
//
//  File name :     assignment40-3.c
//  Descreption :   write a recursive program which display below pattern.
//  Author :        Vaishanvi D Shingare
//  Date :          28/04/2025
//
//  output  :   5  4   3  2  1
///////////////////////////////////////////////////////

void Display() 
{
    static int i = 5; 
    if (i >= 1) 
    {     
        printf("%d\t", i); 
        i--;              
        Display();        
    }
}

int main() {
    Display();     
    return 0;
}